cfme/common/provider.py

```diff
@@ -582,6 +582,7 @@ def get_paginator_value():
 class CloudInfraProvider(BaseProvider, PolicyProfileAssignable):
     vm_name = ""
     template_name = ""
+    volume_name = "Cloud volumes"
```

Comment:
Agreed - This looks like it should be in Cloud, and it looks like the intent of the function using it is solely to navigate to Cloud Volumes via the details page for a specific provider.

I do think it would be best to represent these pages as destinations, though it may mean many destinations for an object. Ideally we want it version picked as well since we've seen the InfoBox rows change, for example 'Images' to 'Container Images'.

If not a navmazing destination, then a load_<name>() function similar to load_details, that uses the base Details destination and then interacts with the InfoBox - again only relevant for Cloud providers for this specific instance.